editor/src/components/editor/actions/actions.tsx

```diff
@@ -1419,7 +1419,7 @@ function toastOnGeneratedElementsTargeted(
   }
 
   if (!generatedElementsTargeted || allowActionRegardless) {
-    result = actionOtherwise(result)
+    result = actionOtherwise(editor)
```

Comment:
I changed this to `editor` from `result` because if `result` is passed in, `generatedElementsTargeted` is false (so it becomes true after the negation), and `allowActionRegardless` is true, both the toast and the result of the action would be added to the resulting editor state, and we only want one of those